src/vs/workbench/contrib/search/browser/searchNotebookHelpers.ts

```diff
@@ -0,0 +1,96 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { FindMatch } from 'vs/editor/common/model';
+import { CellFindMatchWithIndex, ICellViewModel, CellWebviewFindMatch } from 'vs/workbench/contrib/notebook/browser/notebookBrowser';
+
+import { ISearchRange, ITextSearchPreviewOptions, TextSearchMatch } from 'vs/workbench/services/search/common/search';
+import { Range } from 'vs/editor/common/core/range';
+
+export interface NotebookMatchInfo {
```

Comment:
The processing in this file (the way that convert to text matches) is the way it is to imitate the way that we get results from binded text models.